python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -971,12 +971,10 @@ def _setFlushed(self):
         if self._schema:
             for k in self._schema.allItemNames():
                 ## Avoid attributes the likes of job.master which crawl back up the tree
-                if self._schema[k].hasProperty('visitable'):
-                    if (not self._schema[k].getProperties()['visitable'] or self._schema[k].getProperties()['visitable'] == 0):
-                        continue
-                if self._schema[k].hasProperty('transient'):
-                    if(self._schema[k].getProperties()['transient'] or self._schema[k].getProperties()['transient'] == 1):
-                        continue
+                if self._schema[k].hasProperty('visitable') and not self._schema[k].getProperties()['visitable']:
+                    continue
+                if self._schema[k].hasProperty('transient') and self._schema[k].getProperties()['transient']:
+                    continue
```

Comment:
This can be simplified further to:
```python
if not self._schema[k]['visitable']:
    continue
if self._schema[k]['transient']:
    continue
```
as there are no schema items without `visitable` or `transient` defined.